src/Lecture1.hs

```diff
@@ -81,7 +85,16 @@ and 1 is the smallest, and 7 - 1 = 6.
 Try to use local variables (either let-in or where) to implement this
 function.
 -}
-minmax x y z = error "TODO"
+minmax :: Int -> Int -> Int -> Int
+minmax x y z =
+    let
+        a = min x y
+        b = min a z
+        c = max x y
+        d = max c z
+    in
+        d - b
```

Comment:
That's correct 👍🏻 
I personally would write this function a bit more differently and would group and name variables also differently for more readability but probably this doesn't matter much for such small function 🙂 

```suggestion
    let
        minVal = min x (min y z)
        maxVal = max x (max y z)
    in
        maxVal - minVal
```